tools/finish_release.py

```diff
@@ -195,9 +186,9 @@ def main(args):
     # again fails. Publishing the snaps can be done multiple times though
     # so we do that first to make it easier to run the script again later
     # if something goes wrong.        
-    if parsed_args.publish_snaps:
+    if version.startswith('1.'):
```

Comment:
Maybe an explanatory comment here to explain the strategy and effect? Without context, the code isn't super clear about what it's doing.